index.xhtml

```diff
@@ -15,96 +15,113 @@ canonical: /
   <h1>{{ page.title }}</h1>
   <nav>
     <h2>Join us!</h2>
-    <a class="shell button reddit"
-       href="https://reddit.com/r/ProgrammingLanguages">Subreddit</a>
-    <a class="shell button irc"
-       href="#join-irc">IRC</a>
-    <a class="shell button discord"
-       href="https://discord.gg/4Kjt3ZE">Discord</a>
+    <a class="shell button prompt-sign reddit" href="https://reddit.com/r/ProgrammingLanguages">Subreddit</a>
+    <a class="shell button prompt-sign irc" href="#join-irc">IRC</a>
+    <a class="shell button prompt-sign discord" href="https://discord.gg/4Kjt3ZE">Discord</a>
   </nav>
 </section>
 
 <!-- Source of the many icons used here: https://github.com/simple-icons/simple-icons -->
 <!-- Exceptions: The Libera, Reddit, IRC, and Discord icons. I don't know where those are from. -->
 <section id="projects">
   <h2>Take a look at our projects</h2>
+  <div id="tags" style="display: none;">
+    {% assign unique_tags = site.data.tags | uniq %}
+    {% for tag_name in unique_tags %}
+    <div class="shell tag" onclick='setFilter(this,"{{tag_name}}")'>{{ tag_name }}</div>
+    {% endfor %}
+    <div class="shell tag" onclick='setFilter(this,null)'>none</div>
+
+  </div>
```

Comment:
`display: none;` does not actually remove the content, it just hides it. It has no effect on browsers which do not understand CSS, such as Lynx. It would be preferable if this content were added by JS in the first place.